README.md

```diff
@@ -39,7 +39,16 @@ plugins: [
             // boolean (defaults to false) on whether you want
             // to include analytics.page() automatically
             // if false, see below on how to track pageviews manually
-            trackPage: false
+            trackPage: false,
+
+            // boolean (defaults to false) on whether to load segment
+            // after a user action (scroll or route change) + delay
+            // this will bring down your TTI but you might miss 1 second of data.
+            // see here for more info on TTI: https://github.com/GoogleChrome/lighthouse/blob/master/docs/scoring.md#performance
+            delayLoad: false,
+
+            // time to wait after scroll action in ms. Defaults to 1000ms
+            delayLoadTime: 1000
```

Comment:
I mean, technically you can, but I wouldn't want to force people to choose their delay time if they just want to 'turn it on'. 1s is a pretty good default and tweaking that to a different number is, in my opinion, a next step. Is this solely to keep the number of options down?

Reply:
correct, solely to keep the number of options down.  don't want to explode this section because as segment and gatsby grow into larger, mature organizations... sections like this will also get bloated.